### ISSUE TITLE
midterm report feedback

### ISSUE THREAD

**Issue by Tdrinker** [ahemedbullo/CS506-Final-Project#2]
in your readme, you should provide instructions on how to run your code and each file is doing. it should also contain visualization of the data instead of in the visualization folder for better readability. moreover, you should report performances of your models and any baselines you are comparing which I did not find in this report.

also you should report all model details like whats the ratio for data splitting? are you splitting into train/test/val or only train/test? if so, any techniques for preventing overfitting to the test set? are you doing cross validation?

also the writeup is really not specific in what you have access to and what you are predicting. At test time, it sounds like you are trying to predict the percent change in sp500. And you already have access to a few key stocks like meta, apple, nvidia? those key stocks already made up a significant portion of the index I believe. so can't you just do a weighted average static calculation and the result should be already close to the SP500 right? why needing machine learning here? My concern here is it looks like we are trying to predict a number that we can already statically calculate. Definitely needs more clarifications here.

Overall: 6/10
